extensions/blocks/wordads/index.js

```diff
@@ -37,7 +37,10 @@ export const settings = {
 		</Fragment>
 	),
 
-	icon,
+	icon: {
+		src: icon,
+		foreground: '#00be28',
```

Comment:
I think you can update that one too.

Reply:
Great catch — updated in https://github.com/Automattic/jetpack/pull/15311/commits/3c4c406d2aebc152cb43348ca07a3d1bae96a7e3